src/pkg/runner/actionrunner.go

```diff
@@ -0,0 +1,17 @@
+package runner
+
+import (
+	"context"
+
+	"github.com/defenseunicorns/zarf/src/pkg/message"
+	zarfUtils "github.com/defenseunicorns/zarf/src/pkg/utils"
+	zarfTypes "github.com/defenseunicorns/zarf/src/types"
+)
+
+// ActionRunner defines the interface for running actions.
+type ActionRunner interface {
```

Comment:
I think the abstraction created by the interface doesn't quite make sense. I think we'll only ever have Zarf action runners implement this interface. Understood this makes mocking in unit tests a bit easier but I don't that's worth the extra abstraction.